static/js/ducks/observations.js

```diff
@@ -43,6 +45,10 @@ export function fetchObservations(filterParams = {}) {
     filterParams.endDate = dayjs().utc().format("YYYY-MM-DDTHH:mm:ssZ");
   }
 
+  if (!Object.keys(filterParams).includes("numPerPage")) {
+    filterParams.numPerPage = 10;
```

Comment:
Please make sure the default is not defined in more places than it should be (i.e., once for API calls, once for front end, at the most).

Reply:
I think this duck is the right place for this one.